tools/demo.rb

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env ruby
+# Seed demo data to H2MS
+# Creates specified users, if they don't already exist, and then seeds a bunch
+# of events to make interesting data to look at.
+#
+# Example usage, seed 1,000 events:
+#   ./demo.rb -s https://test.h2ms.org/ -u 'admin@h2ms.org' -p 'password' -e 1000
+#
+
+require 'json'
+require 'net/http'
+require 'optparse'
+require 'time'
+require 'uri'
+
+OAUTH_ID = 'h2ms'.freeze
+OAUTH_SECRET = 'secret'.freeze
+
+@locations = [
+  {name: 'Massachusetts General Hospital', address: '55 Fruit St, Boston, MA', zip: '02114', type: 'Hospital' },
+  {name: 'MGH Emergency Room', address: '55 Fruit St, Boston, MA', zip: '02114', type: 'ER', parent: 'Massachusetts General Hospital' },
+  {name: 'MGH Intensive Care Unit', address: '55 Fruit St, Boston, MA', zip: '02114', type: 'ICU', parent: 'Massachusetts General Hospital'},
+
+  {name: 'Sturdy Memorial Hospital', address: '211 Park St, Attleboor, MA', zip: '02703', type: 'Hospital' },
+  {name: 'Sturdy Emergency Room', address: '211 Park St, Attleboor, MA', zip: '02703', type: 'ER', parent: 'Sturdy Memorial Hospital' },
+  {name: 'Sturdy Intensive Care Unit', address: '211 Park St, Attleboor, MA', zip: '02703', type: 'ICU', parent: 'Sturdy Memorial Hospital' },
+]
+
+@users = [
+  { firstName: 'Ben', lastName: 'Jenkins', email: 'bjenkins@h2ms.org', roles: 'ROLE_OBSERVER', password: 'password', type: 'Infection Prevention Nurse' },
+  { firstName: 'Wes', lastName: 'Skillern', email: 'wskillern@h2ms.org', roles: 'ROLE_USER', password: 'password', type: 'ER Doctor' }
```

Comment:
Can we change these to `observer@h2ms.org` and `user@h2ms.org`?

With the first name `Default` and last name's `Observer` and `User`?

I think it would be easier to remember and is consistent with the default admin user.